packages/fluentui/docs/src/components/ComponentPlayground/ComponentPlaygroundSnippet.tsx

```diff
@@ -11,6 +11,7 @@ type ComponentPlaygroundSnippetProps = {
  */
 const ComponentPlaygroundSnippet: React.FunctionComponent<ComponentPlaygroundSnippetProps> = props => {
   const { element, component, ...rest } = props;
+  return null;
```

Comment:
Probably want to revert this.

Reply:
Yeap, haha 😓. Overzealous `git add` while breaking up commits.